src/tracking_state.jl

```diff
@@ -24,6 +24,24 @@ end
 """
 $(SIGNATURES)
 
+CarrierReplicaCPU that holds possible carrier representations.
+The type is unknown, when TrackingState is initialized.
+It is determined based on the signal, which is given in the track
+function.
+"""
+struct CarrierReplicaGPU{
```

Comment:
The CarrierReplicaGPU is never used, isn't it? I guess we can get rid of this structure.